xbmc/cores/RetroPlayer/shaders/gl/ShaderGL.cpp

```diff
@@ -50,7 +50,7 @@ bool CShaderGL::Create(const std::string& shaderSource, const std::string& shade
   else
     defineFragment = "#define FRAGMENT\n#define PARAMETER_UNIFORM\n";
 
-  if (m_shaderSource.rfind("#version", 0) == 0)
+  if (m_shaderSource.rfind("#version", 0) == std::string::npos)
```

Comment:
This change completely breaks functionality

Reply:
I misread the code. This change is incorrect.